intellij-plugin-verifier/verifier-core/src/main/java/com/jetbrains/pluginverifier/verifiers/filter/SameModuleUsageFilter.kt

```diff
@@ -0,0 +1,40 @@
+package com.jetbrains.pluginverifier.verifiers.filter
+
+import com.jetbrains.pluginverifier.verifiers.VerificationContext
+import com.jetbrains.pluginverifier.verifiers.hasAnnotation
+import com.jetbrains.pluginverifier.verifiers.resolution.BinaryClassName
+import com.jetbrains.pluginverifier.verifiers.resolution.ClassFileMember
+import com.jetbrains.pluginverifier.verifiers.resolution.Method
+import com.jetbrains.pluginverifier.verifiers.resolution.searchParentOverrides
+import org.objectweb.asm.tree.AbstractInsnNode
+
+private const val DISABLE_SAME_MODULE_INVOCATIONS: BinaryClassName = "com/jetbrains/pluginverifier/verifiers/resolution/DisableSameModuleInvocations"
```

Comment:
It would be better to not have test-environment-only annotations present and handled in production code. Maybe some environment parameter instead set for corresponding test(s) explicitly?

Reply:
Resolved via effective element that declares the corresponding annotation. Opt-out annotation has been removed.